src/lib/core/backtrace.cc

```diff
@@ -186,7 +186,15 @@ backtrace(char *start, size_t size)
  * @param stack pointer to preserve.
  * @retval preserved stack pointer.
  */
-static void *
+#ifdef __aarch64__
+/* On arm64 this function must be declared as extern "C"
+ * in order to be called from asm fragment in in coro_unwcontext().
+ */
+extern "C"
+#else
+static
+#endif
+void *
```

Comment:
Seems that you may call `unw_getcontext_f` from asm code without this change.
See comments below.

Reply:
You were right!
Suggested solution works like a charm.